config/sys.config

```diff
@@ -1,5 +1,10 @@
 
 [
+  { kernel,
+    [
+     {inet_dist_use_interface, {127,0,0,1}}
```

Comment:
On f3b537f53d55a21d02b8de37a59e4bb41a5db280 I conducted this test:

Package built by `make prod-package`, installed&configured&started as per instructions referred to by release notes.

I checked address-port:
```
epmd -names
epmd: up and running on port 4369 with data:
name epoch at port 62480
...
netstat -atn | less
...
tcp4       0      0  127.0.0.1.62480        *.*                    LISTEN
```